x/projects/keeper/project_test.go

```diff
@@ -221,19 +280,22 @@ func SetPolicyTest(t *testing.T, testAdminPolicy bool) {
 	err := keepers.Projects.CreateProject(sdk.UnwrapSDKContext(ctx), subAccount.Addr.String(), projectData, plan)
 	require.Nil(t, err)
 
-	keepers.Projects.AddKeysToProject(sdk.UnwrapSDKContext(ctx), projectID, adminAcc.Addr.String(),
+	err = keepers.Projects.AddKeysToProject(sdk.UnwrapSDKContext(ctx), projectID, adminAcc.Addr.String(),
```

Comment:
1. done
2. I did see that in your code you use _ctx. But this is problematic when you need to update the context in your test (like when you advance an epoch) since _ctx is created in the beginning of the code, before any context changes. Since this test advances blocks, I won't use _ctx

Reply:
as you may see in the code, it re-assigns "ctx" when "_ctx" is updated, as needed.
in fact, I would love the testutils to have this convention all over, and have the functions return both "_ctx" and "ctx"!